validphys2/src/validphys/theorycovariance/output.py

```diff
@@ -270,7 +270,7 @@ def plot_thblockcorrmat_heatmap(theory_blockcorrmat):
 def plot_thcorrmat_heatmap_custom(
     theory_corrmat_custom,
     theoryids,
-    fivetheories: (str, type(None)) = None,
+    fivetheories,
```

Comment:
Same as the previous comment, but for this file you can just run `black` on it. `config.py` is big and modified by many people, this one will be mostly worked by you in the future so it's fine.